src/VsixV3/EditorsPackage/Microsoft.VisualStudio.Editors.swr

```diff
@@ -1,7 +1,7 @@
 use vs
 
 package name=Microsoft.VisualStudio.Editors
-        version=15.0.0.0
+        version=$(Version)
```

Comment:
@natidea you're probably right.  I haven't seen @basoundr this afternoon, so I'll just change this to a hard-coded ```15.0.0.1``` to be safe.